### ISSUE TITLE
Cluster autoscaling e2e tests are flaky

### ISSUE THREAD

**Issue by piosz** [kubernetes/kubernetes#14737]
```
Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on memory utilization

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/autoscaling.go:87
Expected error:
    <*errors.errorString | 0xc2083f3d90>: {
        s: "timeout waiting 20m0s for cluster size to be 3",
    }
    timeout waiting 20m0s for cluster size to be 3
not to have occurred

Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on cpu utilization

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/autoscaling.go:65
Expected error:
    <*errors.errorString | 0xc2083159d0>: {
        s: "Only 5 pods started out of 6",
    }
    Only 5 pods started out of 6
not to have occurred
```

**Comment by goltermann** [comment]
Is this fixed via the merged PR?

**Comment by piosz** [comment]
There are two issues: one is fixed by #14740 and another is fixed by https://github.com/kubernetes/heapster/pull/623, however it requires also new release of Heapster which hasn't been done yet so that I haven't closed this issue.

**Comment by piosz** [comment]
related PRs #14654, https://github.com/kubernetes/heapster/issues/623

**Comment by a-robinson** [comment]
It looks like you're on top of this, but it's been failing pretty frequently this morning as well, all on "Horizontal pod autoscaling [Skipped][Autoscaling Suite] should scale from 5 pods to 3 pods and from 3 to 1 (scale resource: Memory)"

**Comment by piosz** [comment]
The issue is about cluster size autoscaling tests, which haven't failed since #14559 was merged. I'll close this issue tomorrow once there will be no more failures. Also there is another issue with Horizontal Pod Autoscaling test - tracking issue #14667

**Comment by piosz** [comment]
The test is stable now so closing the issue.

**Comment by jszczepkowski** [comment]
This problem is occurring again:
http://kubekins.dls.corp.google.com/view/All/job/kubernetes-e2e-gce-autoscaling/1293/

**Comment by lavalamp** [comment]
```
Horizontal pod autoscaling [Skipped][Autoscaling Suite] should scale from 5 pods to 3 pods and from 3 to 1 (scale resource: CPU)

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/horizontal_pod_autoscaling.go:57
Oct 26 05:30:49.950: Number of replicas has changed: expected 3, got 2
```

blocking merges; raising to P0

**Comment by lavalamp** [comment]
looks like https://github.com/kubernetes/kubernetes/pull/16274 was merged to hopefully address this. Lowering to p1.

**Comment by piosz** [comment]
Yep, #16274 disabled the test, tracking issue #16275 and there is fix in flight #16277

**Comment by jszczepkowski** [comment]
Closing as #16275 is resolved.
